dev/tools/java_and_objc_doc.dart

```diff
@@ -23,9 +23,23 @@ Future<void> main(List<String> args) async {
 }
 
 Future<void> generateDocs(String url, String docName, String checkFile) async {
-  final http.Response response = await http.get(url);
+  const int maxTries = 5;
+
+  List<int> responseBytes;
```

Comment:
I was going to put this whole block of code into a separate function that returns an Archive, but I guess it's just for prettiness, since it's only called here.

Reply:
Done. Just returning null on failure rather than dealing with an exception since this is effectively one-off utility code.

Reply:
Also added a tiny bit of code just now to print out the first 80 chars of the body in case it turns out to be useful in diagnosing what's up (on the assumption that if we're not getting a 200, we may be getting an error msg.